src/components/MAPI/permissions/utils.ts

```diff
@@ -521,6 +521,51 @@ export function hasAppAccess(
   );
 }
 
+export function hasAccessToInspectPermissions(
+  permissions: IPermissionMap
+): boolean {
+  switch (true) {
+    case hasPermission(
+      permissions,
+      '',
+      'list',
+      'rbac.authorization.k8s.io',
+      'clusterroles'
+    ):
+    case hasPermission(
+      permissions,
+      '',
+      'list',
+      'rbac.authorization.k8s.io',
+      'clusterrolebindings'
+    ):
+    case hasPermission(
+      permissions,
+      '',
+      'list',
+      'rbac.authorization.k8s.io',
+      'roles'
+    ):
+    case hasPermission(
+      permissions,
+      '',
+      'list',
+      'rbac.authorization.k8s.io',
+      'rolebindings'
+    ):
+    case hasPermission(
+      permissions,
+      'default',
+      'create',
+      'authorization.k8s.io',
+      'localsubjectaccessreviews'
+    ):
+      return true;
```

Comment:
Just to ensure, it should return `true` if any one of these `hasPermission` calls returns `true`, not all of them, right?

Reply:
That's a really good point - it should return `true` only if all the conditions are met 😬 Thanks for keeping a sharp eye!